src/main.nqp

```diff
@@ -14,8 +17,8 @@ $comp.parsegrammar(Perl6::Grammar);
 $comp.parseactions(Perl6::Actions);
 $comp.addstage('syntaxcheck', :before<ast>);
 $comp.addstage('optimize', :after<ast>);
-hll-config($comp.config);
-nqp::bindhllsym('Raku', '$COMPILER_CONFIG', $comp.config);
+hll-config(nqp::gethllsym('default', 'SysConfig').rakudo-build-config);
+nqp::bindhllsym('Raku', '$COMPILER_CONFIG', nqp::gethllsym('default', 'SysConfig').rakudo-build-config);
```

Comment:
Similar note as for `config()` method name. Except that here a name like `$SYSTEM_CONFIG` fits even better than `$COMPILER_SYSCONFIG`.

Reply:
I removed `$COMPILER_SYSCONFIG` entirely. It was only used in a few places.